e2e/main/main.po.ts

```diff
@@ -1,16 +1,58 @@
-import { by, element } from 'protractor';
+import { by, element, ElementFinder } from 'protractor';
 import { BasePage } from '../common/components/base-page';
 
 export class MainPage extends BasePage {
-	constructor() {
-		super('systelab-app-frame');
-	}
+    constructor() {
+        super('systelab-app-frame');
+    }
 
-	public getFullUsernameField() {
-		return this.getObjectById('username');
-	}
+    public getFullUsernameField() {
+        return this.getObjectById('username');
+    }
 
-	public getPatientButtton() {
-		return this.getMainWindow().element(by.tagName('systelab-app-header')).all(by.tagName('button')).get(0);
-	}
+    public getPatientButton() {
+        return this.getMainWindow().element(by.tagName('systelab-app-header')).all(by.tagName('button')).get(0);
+    }
+
+    public getConfigIcon() {
+        return this.getMainWindow().element(by.tagName('systelab-app-sidebar-small')).all(by.tagName('li')).get(3);
+    }
+
+    public getAllergyOptionsButton() {
```

Comment:
The file "main.po.ts" must contain only the access to the components in the main frame. The access to the patient allergies tab must be in a specific PO for it.

Reply:
The allergy management screen is part of the main frame, as it is accessed from the sidebar and no new page is created/displayed.
![image](https://user-images.githubusercontent.com/39374121/64119888-28399c80-cd9b-11e9-939c-1b8bffda7526.png)